core/experiments/manager.php

```diff
@@ -440,8 +440,7 @@ private function add_default_features() {
 			'title' => esc_html__( 'Optimized Markup', 'elementor' ),
 			'tag' => esc_html__( 'Performance', 'elementor' ),
 			'description' => esc_html__( 'Reduce the DOM size by eliminating HTML tags in various elements and widgets. This experiment includes markup changes so it might require updating custom CSS/JS code and cause compatibility issues with third party plugins.', 'elementor' ),
-			static::TYPE_HIDDEN => true,
-			'release_status' => self::RELEASE_STATUS_DEV,
+			'release_status' => self::RELEASE_STATUS_BETA,
```

Comment:
The plugin tests are currently failing. I am not sure if that is related to your PR.